repository/sparql_reader.go

```diff
@@ -14,6 +15,9 @@ import (
 const tripleLimit = 9000
 const idVariable = "?id"
 
+// Number of times to retry connecting to the triplestore
+const retries = 180
```

Comment:
Retrying 180 times seems like a lot? Have you seen specific error patterns that warrant a high number like this?